core/src/mindustry/ui/dialogs/ModsDialog.java

```diff
@@ -136,14 +139,12 @@ void setup(){
         cont.row();
 
         if(!mods.list().isEmpty()){
-            cont.pane(table -> {
-                table.margin(10f).top();
-
-                boolean anyDisabled = false;
-                for(LoadedMod mod : mods.list()){
-
-                    if(!mod.enabled() && !anyDisabled && mods.list().size > 0){
-                        anyDisabled = true;
+            boolean[] anyDisabled = {false};
+            SearchBar.add(cont, mods.list(),
+                mod -> Strings.stripColors(mod.meta.displayName()),
```

Comment:
The display name already has colors stripped.

Reply:
yes its an old pr